### ISSUE TITLE
Styling Projects Page

### ISSUE THREAD

**Issue by AnirudhPanda** [Opentek-Org/opentek#13]
The cards design looks a bit pale. A little more animated or UI redesign is needed.

**Comment by neeraj15022001** [comment]
![Screenshot 2021-09-13 at 11 36 47 PM](https://user-images.githubusercontent.com/55191873/133134431-1165fca6-6e1d-4971-9c57-8bc874c4f6d6.png)
Here i think they will look good after some color changes and height adjustments and i can intergrate tilt.js alongside

**Comment by neeraj15022001** [comment]
i would like to work on this

**Comment by AnirudhPanda** [comment]
Yeah using tilt.js would look great!
Make sure it is minimalistic though!

Assigning to you!

**Comment by neeraj15022001** [comment]
![MacBook - 1](https://user-images.githubusercontent.com/55191873/133139812-8991efb4-1ebb-400e-81c4-dbfb5b9be276.png)
How about this design ?

**Comment by AnirudhPanda** [comment]
> ![MacBook - 1](https://user-images.githubusercontent.com/55191873/133139812-8991efb4-1ebb-400e-81c4-dbfb5b9be276.png)
> How about this design ?

Looks Great!!

Can you do something that would keep the entire project's name?
If half of the name comes down to another line that's not an issue but if due to that the whole styling (card) gets on another line it would not align with the other cards

**Comment by neeraj15022001** [comment]
Yes, to overcome this problem we will have to trim text after a particular length like showed in above design. 
if we will scale down text size it will impact UX. We can do one thing that we will add tooltip on Title for example on hovering on title whole title will show up in tooltip or another option is to make text overflow in x direction with a scroll
but i personally feel tootip is god option

**Comment by AnirudhPanda** [comment]
> Yes, to overcome this problem we will have to trim text after a particular length like showed in above design.
> if we will scale down text size it will impact UX. We can do one thing that we will add tooltip on Title for example on hovering on title whole title will show up in tooltip or another option is to make text overflow in x direction with a scroll
> but i personally feel tootip is god option

Yeah we can go with tooltip and maybe decrease font size a bit

**Comment by neeraj15022001** [comment]
> > Yes, to overcome this problem we will have to trim text after a particular length like showed in above design.
> > if we will scale down text size it will impact UX. We can do one thing that we will add tooltip on Title for example on hovering on title whole title will show up in tooltip or another option is to make text overflow in x direction with a scroll
> > but i personally feel tootip is god option
> 
> Yeah we can go with tooltip and maybe decrease font size a bit

According to me decreasing font size will destroy the purpose of Title aa Title should be the first catchy thing for user as we ourselves when visit a website scan it ans not read details . So Title should be large enough to catch user's eye. Rest if you want to discuss with others than we can discuss with other about their review on this

**Comment by neeraj15022001** [comment]
Shall i increase card height and width, it will solve the issue you were talking about earlier

**Comment by AnirudhPanda** [comment]
> Shall i increase card height and width, it will solve the issue you were talking about earlier

You can try if it works than great

**Comment by nawed2611** [comment]
Hey! Is this issue still open? I wanted to work on it.

**Comment by shraddha761** [comment]
Hye request to assign this issue to me under GSSOC'23
